docs/dev/README.md

```diff
@@ -0,0 +1,102 @@
+# Quick start guide for Assisted Service Developer
+
+Over time, assisted-service has been adapted to different scenarios, and has become one more piece of many other subsystems in which it works integrated with other Openshift components.
+
+For this reason, depending on the development we are working on, we have different scenarios for developers that allow us greater flexibility for our daily work. For example, there will be cases in which we are only interested in doing some specific tests on the API and this can be covered with a light environment, but in other cases we may need a complete Openshift cluster with its control plane and worker nodes, in which case the scenario will be more complex and heavy.
+
+The frequently used scenarios are described below.
+
+| tool | resource usage | run it locally | useful for |
+| :---: | :---: | :---: | :---: |
+| [podman](##podman) | light | :white_check_mark: | small validations of API REST service |
+| [crc](##crc) | medium | :white_check_mark: | tests with a local SNO Openshift/Microshift cluster |
+| [assisted-test-infra](##assisted-test-infra) | heavy | :x: | testing and development of end-to-end tests |
+| [infrastructure operator](#infrastructure-operator) | light / heavy | :x: | this should be valid to any test related with assisted-installer |
```

Comment:
why is infrastrcuture-operator interleaved with podman,crc,test-infra and dev-scripts? Infrastructure Operator is a way to deploy assisted, while the others are frameworks/platforms

Reply:
The orientation, in all cases, is to do _something_ related to assisted. Maybe it is not clear when putting `podman` or `crc` in other sections, in the case of infrastructure operator it would be `kind`.

Would it be clearer for a new developer if we put `kind` instead of `infrastructure operator`?

Reply:
Yes I think we should rename the table entry/link to match the paragraph below